gymnasium/envs/mujoco/mujoco_rendering.py

```diff
@@ -611,7 +611,7 @@ def _create_overlay(self):
 
         self.add_overlay(bottomleft, "FPS", "%d%s" % (1 / self._time_per_render, ""))
         self.add_overlay(
```

Comment:
It is possible to maintain support for both pre 3.0 and post 3.0 `mujoco` (which is important for reproducibility), by doing this

```py
if mujoco.__version__ => "3.0.0":
    self.add_overlay(
        bottomleft, "Solver iterations", str(self.data.solver_niter[0] + 1)
    )
elif mujoco.__version__ < "3.0.0":
    self.add_overlay(
        bottomleft, "Solver iterations", str(self.data.solver_iter + 1)
    )
```